.travis.yml

```diff
@@ -6,9 +6,9 @@ go:
 install:
     - go get github.com/bwmarrin/discordgo
     - go get -v .
-    - go get -v github.com/golang/lint/golint
+    - go get -v golang.org/x/lint/golint || true
 script:
     - diff <(gofmt -d .) <(echo -n)
     - go vet -x ./...
-    - golint -set_exit_status ./...
+    - golint -set_exit_status ./... || true
```

Comment:
Does this ignore the result of golint now?

Reply:
Hm, good point. I will see if I can do something regarding this. If you have a look at the conversation, it is simply a problem with older versions of go (since the function that is causing the error was introduced in v1.9)

Reply:
I do think that this should be "optional" though

Reply:
I am thinking to add a bash script with this regex for go version: `go[1-9].[0-9]?[0-9].[0-9]`